### DISCUSSION TITLE
ReplayGain Support

### DISCUSSION THREAD

**Discussion by oliveoil1** [rekkyrosso/ampcast#8] [Ideas]
Hi, loving the project so far.

Is there any plans to add support for adjusting song volume based their ReplayGain tags? I think most of the self hosted providers currently support this so it would be nice to see this implemented. 

Thanks.

**Comment by rekkyrosso** [comment]
I will look into it.

**Comment by rekkyrosso** [comment]
Added in version 0.9.10.

Available via Settings>Personal Media>Audio
